cli/template/base/src/env/schema.mjs

```diff
@@ -6,7 +6,7 @@ import { z } from "zod";
  * This way you can ensure the app isn't built with invalid env vars.
  */
 export const serverSchema = z.object({
-  // SERVERVAR: z.string(),
+  NODE_ENV: z.enum(["development", "test", "production"]),
```

Comment:
Nice.

We also need it in the auth-schema:

https://github.com/t3-oss/create-t3-app/blob/ff64216a89261474dee966272a8e49ffd0b1b9a6/cli/template/addons/env/auth-schema.mjs#L8-L13

Reply:
Done